### ISSUE TITLE
Dashboard colour usage

### ISSUE THREAD

**Issue by JVKE93** [DFE-Digital/accessibility-issues-statements#44]
Issue 5: Use of colour - Error
Location: [Reports](https://accessibility-statements.education.gov.uk/reports)
Severity: Medium

In the reports view of the open issues a RAG status is applied to show an increase in numbers within a certain WCAG criterion. Colour alone has been used to represent the change in RAG status.
Users with a colour deficiency may be unable to perceive colours used within this report. They could therefore miss out on context provided by these colours
Recommendation: Further context could be added to show the change in RAG status. A symbol with a key, or R, A, G, could be provided a corner of the square.

<img width="274" alt="Image" src="https://github.com/user-attachments/assets/1cd62532-3386-4f64-b788-dd0b2b49e32d" />
